### PR TITLE
See commit messages for details

### PR DESCRIPTION
**Pull request by ash663**
See commit messages for details.

Testing:

> Ran the bot on a private server

> Executed a bunch of commands, posted screenshots in #rubber-ducky

### CODE REVIEW

**Review by ingenium21 — APPROVED** [review]
This works okay.  !anySignings appears to be broken but that's not your fault.

File: requirements.txt [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-pandas
+discord~=1.7.3
+bs4~=0.0.1
+datetime~=4.3
+tabulate~=0.8.9
+Pillow~=9.0.0
+pandas~=1.4.0
+
+requests~=2.27.1
+beautifulsoup4~=4.10.0
```
Comment (AndyReifman) [inline requirements.txt:9]: Is there a reason we've decided to specify versions?

Comment (ash663) [reply requirements.txt:9]: General hygiene for dependency management. This will ensure build are easily reproducible in any environment, and predictable.

Comment (AndyReifman) [reply requirements.txt:9]: Good Answer. ⭐

**Review by AndyReifman — APPROVED** [review]

**Pull request merged**